onnxscript/rewriter/ort_fusions/shape_optimization.py

```diff
@@ -55,7 +55,7 @@ def rewrite(self, op, dim0, dim1, dim2, dim3, **_):
         transposed_dims = [dim0, dim2, dim1, dim3]
         sliced_result = transposed_dims[self._start_val : self._end_val]
         if len(sliced_result) == 0:
-            return op.Constant(value_ints=[])
+            return op.Constant(value_ints=ir.AttrInt64s("value_ints", []))
```

Comment:
This makes it inconsistent with onnxscript, where we know the attribute type already from the schema. I wonder if there is a way to bridge this?